### PR TITLE
RUM-718 docs: update instructions to perform URLSession instrumentation

### PR DESCRIPTION
**Pull request by ganeshnj**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by ganeshnj** [comment]
> Some small comments.
> 
> Just curious if you used some particular configuration or linting that changed end lines?

remove trailing whitespaces.

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 1]
```diff
@@ -180,11 +180,20 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
     )
     ```
 
-4. Initialize URLSession as stated in [Setup][1]:
+4. Enable URLSession instrumentation with your `URLSessionDataDelegate` implementation:
```
[the remark this answers is not in the document]
Comment (ganeshnj) [reply content/en/real_user_monitoring/connect_rum_and_traces/_index.md:183]: reworded to make obvious

**Comment by aliciascott** [comment]
Thank you, I have added Editorial Review card for the Docs team: https://datadoghq.atlassian.net/browse/DOCS-6289

**Review by jhgilbert — APPROVED** [review]
Approved with very minor changes for style, thank you!

**Pull request merged**